### DISCUSSION TITLE
About footer of homepage

### DISCUSSION THREAD

**Discussion by momo91377** [web-infra-dev/rspress#1396] [Q&A] [answered]
各位老师好，想请教一下当前的footer是否支持设置跳转链接形式，这里我按照官方用户手册的说明
<img width="785" alt="Snipaste_2024-09-06_11-49-06" src="https://github.com/user-attachments/assets/6340405a-9824-448f-9a07-2a6debd96886">
进行了配置：
![image](https://github.com/user-attachments/assets/fc459343-cf7c-4214-b600-1db203cae323)
但是生成效果还是展示为如下纯文本形式：
![image](https://github.com/user-attachments/assets/86286e09-7455-4d0f-b97b-f5736f60bb7e)
请教一下除此项设置，是否是还需要进行其他配置配合使用才能生效？

**Comment by Timeless0911** [comment, marked as answer]
版本太低了，https://github.com/web-infra-dev/rspress/pull/1240 已经支持了

**Reply by momo91377** [reply]
了解，我升级版本做下尝试

**Reply by momo91377** [reply]
请问这个对应的是rspress哪个版本呢

**Reply by Timeless0911** [reply]
你可以直接看 release 说明，建议直接最新

**Reply by momo91377** [reply]
Get，我先升级最新做下尝试

**Reply by momo91377** [reply]
请问这个必须是生产环境么，本地开发环境已升级至最新版本1.29.0
![image](https://github.com/user-attachments/assets/bc3ff4df-c2ec-4edc-8753-85af69d9ce53)
但是效果还是...
![image](https://github.com/user-attachments/assets/eec3c20d-0199-441a-b90b-57c14a5b7eca)

**Reply by Timeless0911** [reply]
我这边没有问题，你可以在 rspress 的仓库里自己测试下 https://github.com/web-infra-dev/rspress/blob/main/e2e/fixtures/custom-home-footer/rspress.config.ts

**Reply by momo91377** [reply]
我按照这里面一样的做了下尝试
![image](https://github.com/user-attachments/assets/86ba0ae2-d555-46a8-a097-f52fa11e8432)
效果如下：
![image](https://github.com/user-attachments/assets/6942f9e1-ae65-4b42-b3cc-48c0cbd34d26)

**Reply by momo91377** [reply]
😥好奇怪,这个会跟什么其他配置冲突么

**Reply by Timeless0911** [reply]
Can you provide a reproduction demo or you should debug on your own.

**Reply by momo91377** [reply]
目前不太好提供demo...我们再做下尝试，十分感谢
